### PR TITLE
Add syntax section for logical replication

### PR DESCRIPTION
**Pull request by dr0pdb**

### CODE REVIEW

File: docs/content/preview/api/ysql/the-sql-language/statements/ddl_alter_publication.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+#### RENAME TO *publication_name*
+
+Rename the publication name to the specified name.
+
+#### OWNER TO *new_owner*
+
+Change the owner of the publication to the *new_owner*.
+
+### Permissions
```
Comment (vaibhav-yb) [inline docs/content/preview/api/ysql/the-sql-language/statements/ddl_alter_publication.md:50]:
```suggestion
### Permissions

```

File: docs/content/preview/api/ysql/the-sql-language/statements/ddl_alter_publication.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+yugabyte=# ALTER PUBLICATION my_publication ADD TABLE users, departments;
+```
+
+Drop tables `employees`from the publication `my_publication`.
+
+```sql
+yugabyte=# ALTER PUBLICATION my_publication DROP TABLE employees;
+```
+
+Set the tables to only `departments`in the publication `my_publication`. The table `users` will be removed from the publication as a result of the command.
```
Comment (vaibhav-yb) [inline docs/content/preview/api/ysql/the-sql-language/statements/ddl_alter_publication.md:73]:
```suggestion
Set the tables to only `departments` in the publication `my_publication`. The table `users` will be removed from the publication as a result of the command.
```

File: docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_publication.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+{{%ebnf%}}
+  create_publication,
+  publication_for_option
+{{%/ebnf%}}
+
+## Semantics
+
+Create a publication with *publication_name*. If `publication_name` already exists in the specified database, an error will be raised.
+
+### Tables
```
Comment (vaibhav-yb) [inline docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_publication.md:28]:
```suggestion
### Tables

```

File: docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_publication.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+The `FOR TABLE` option specifies a list of tables to add to the publication while the `FOR ALL TABLES` option marks the publication as one that replicates changes for all tables in the database, including tables created in the future.
+
+If `FOR TABLE` or `FOR ALL TABLES` are not specified, then the publication starts out with an empty set of tables. That is useful if tables are to be added later.
```
Comment (vaibhav-yb) [inline docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_publication.md:31]:
```suggestion
If `FOR TABLE` or `FOR ALL TABLES` are not specified, then the publication starts out with an empty set of tables. This is useful if tables are to be added later.
```

File: docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_publication.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+### Permissions
```
Comment (vaibhav-yb) [inline docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_publication.md:33]:
```suggestion
### Permissions

```

File: docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_publication.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+To create a publication, the invoking user must have the `CREATE` privilege for the current database. (Superusers bypass this check.)
+
+To add a table to a publication, the invoking user must have ownership rights on the table. The FOR ALL TABLES clause requires the invoking user to be a **superuser**.
```
Comment (vaibhav-yb) [inline docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_publication.md:36]:
```suggestion
To add a table to a publication, the invoking user must have ownership rights on the table. The `FOR ALL TABLES` clause requires the invoking user to be a **superuser**.
```

File: docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_publication.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+### Limitations
```
Comment (vaibhav-yb) [inline docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_publication.md:38]:
```suggestion
### Limitations

```

File: docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_publication.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+Currently to publish a subset of operations (create, update, delete, truncate) via a Publication is not supported.
+
+## Examples
+
+### Publication with two tables
+
+```plpgsql
```
Comment (vaibhav-yb) [inline docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_publication.md:45]: Any specific reason to specify `plpgsql` here as compared to `sql` in other places?

File: docs/content/preview/api/ysql/the-sql-language/statements/streaming_create_repl_slot.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+### *output_plugin*
+
+The name of the output plugin used for logical decoding.
+
+### NOEXPORT_SNAPSHOT / USE_SNAPSHOT
+
+Decides what to do with the snapshot created during logical slot initialization.
+
+USE_SNAPSHOT will use the snapshot for the current transaction executing the command. This option must be used in a transaction, and CREATE_REPLICATION_SLOT must be the first command run in that transaction.
```
Comment (vaibhav-yb) [inline docs/content/preview/api/ysql/the-sql-language/statements/streaming_create_repl_slot.md:37]:
```suggestion
`USE_SNAPSHOT` will use the snapshot for the current transaction executing the command. This option must be used in a transaction, and `CREATE_REPLICATION_SLOT` must be the first command run in that transaction.
```

File: docs/content/preview/api/ysql/the-sql-language/statements/streaming_create_repl_slot.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+
+NOEXPORT_SNAPSHOT will just use the snapshot for logical decoding as normal but won't do anything else with it.
```
Comment (vaibhav-yb) [inline docs/content/preview/api/ysql/the-sql-language/statements/streaming_create_repl_slot.md:39]:
```suggestion
`NOEXPORT_SNAPSHOT` will just use the snapshot for logical decoding as normal but won't do anything else with it.
```

File: docs/content/preview/api/ysql/the-sql-language/statements/streaming_create_repl_slot.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+
+## Examples
+
+Create a Replication Slot with name test_replication_slot and use the yboutput plugin.
```
Comment (vaibhav-yb) [inline docs/content/preview/api/ysql/the-sql-language/statements/streaming_create_repl_slot.md:43]:
```suggestion
Create a Replication Slot with name `test_replication_slot` and use the `yboutput` plugin.
```

File: docs/content/preview/api/ysql/the-sql-language/statements/streaming_create_repl_slot.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+
+```plpgsql
+yugabyte=# CREATE_REPLICATION_SLOT test_replication_slot LOGICAL yboutput;
+```
+
+Create a Replication Slot with name test_replication_slot and use the pgoutput plugin.
```
Comment (vaibhav-yb) [inline docs/content/preview/api/ysql/the-sql-language/statements/streaming_create_repl_slot.md:49]:
```suggestion
Create a Replication Slot with name `test_replication_slot` and use the `pgoutput` plugin.
```

File: docs/content/preview/api/ysql/the-sql-language/statements/streaming_start_replication.md [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+  replication_option
+{{%/ebnf%}}
+
+## Semantics
+
+Instructs the server to start streaming WAL for logical replication. The output plugin associated with the selected slot is used to process the output for streaming.
+
+### *slot_name*
+
+The name of the slot to stream changes from. This parameter is required, and must correspond to an existing logical replication slot created with CREATE_REPLICATION_SLOT.
```
Comment (vaibhav-yb) [inline docs/content/preview/api/ysql/the-sql-language/statements/streaming_start_replication.md:31]:
```suggestion
The name of the slot to stream changes from. This parameter is required, and must correspond to an existing logical replication slot created with `CREATE_REPLICATION_SLOT`.
```

Comment (vaibhav-yb) [reply docs/content/preview/api/ysql/the-sql-language/statements/streaming_start_replication.md:31]: Please link the `CREATE_REPLICATION_SLOT` to its page.

**Review by vaibhav-yb — CHANGES_REQUESTED** [review]

File: docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_publication.md (hunk 8 above)
Comment (dr0pdb) [reply docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_publication.md:45]: No, just copy pasted. I think both works. UPdating to sql for consistency.

**Pull request closed without merging**